controllers/mysqlcluster_controller.go

```diff
@@ -216,6 +227,10 @@ func (r *MySQLClusterReconciler) createPasswordSecretForUser(ctx context.Context
 			return err
 		}
 	}
+	miscPass, err := generateRandomBytes(passwordBytes)
```

Comment:
This is not a password for _users_.

Reply:
Yes, you're right. A root password is the only one that worth checking by users. However, all passwords in the Secret are referred by the entrypoint when it creates MySQL users during the initializing process. So the Secret contains those passwords besides a root password. It's probably better to change the function name to a more appropriate one.

Reply:
How about just moving this to `r.createPasswordSecretForController`?
Are there some obstacles?

Reply:
Currently, we cannot move those passwords to `r.createPasswordSecretForController`, because all passwords in the Secret are referred by the entrypoint when it creates MySQL users during the initializing process.
The entrypoint cannot refer to the Secret in other namespace.

Reply:
I see.  So, how about renaming this method to something like `createPasswordSecretForInit`
because the Secret is used primarily for initialization.